hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/StorageVolume.java

```diff
@@ -382,6 +383,7 @@ public abstract static class Builder<T extends Builder<T>> {
     private boolean failedVolume = false;
     private String datanodeUuid;
     private String clusterID;
+    private long failureDate;
```

Comment:
Lets use `failureTime`. I'm assuming this is being stored as millis since epoch, so it will have data and time information.

Reply:
I have improved the relevant code.